README.md

```diff
@@ -577,50 +508,54 @@ alternative method to pass args if you have 0.9.8.x and below.
 ## Reference
 
 ### Classes
+
 #### Public classes
+
 * [`chocolatey`](#class-chocolatey)
 
 #### Private classes
+
 * `chocolatey::install.pp`: Ensures Chocolatey is installed.
 * `chocolatey::config.pp`: Ensures Chocolatey is configured.
 
 ### Facts
-* `chocolateyversion` - The version of the installed choco client (could also be provided by class parameter `chocolatey_version`).
-* `choco_install_path` - The location of the installed choco client (could also be provided by class parameter `choco_install_location`).
+
+* `chocolateyversion` - The version of the installed Chocolatey client (could also be provided by class parameter `chocolatey_version`).
+* `choco_install_path` - The location of the installed Chocolatey client (could also be provided by class parameter `choco_install_location`).
 
 ### Types/Providers
+
 * [Chocolatey provider](#package-provider-chocolatey)
 * [Chocolatey source configuration](#chocolateysource)
 * [Chocolatey feature configuration](#chocolateyfeature)
 
 
-### Package Provider: Chocolatey
+### Package provider: Chocolatey
+
 Chocolatey implements a [package type](http://docs.puppet.com/references/latest/type.html#package) with a resource provider, which is built into Puppet.
 
 This provider supports the `install_options` and `uninstall_options` attributes,
-which allow command-line options to be passed to the choco command. These options
+which allow command-line options to be passed to the `choco` command. These options
 should be specified as documented below.
 
  * Required binaries: `choco.exe`, usually found in `C:\Program Data\chocolatey\bin\choco.exe`.
-   * The binary is searched for using the Environment Variable `ChocolateyInstall`, then by two known locations (`C:\Chocolatey\bin\choco.exe` and `C:\ProgramData\chocolatey\bin\choco.exe`).
-   * On Windows 2003 you should install Chocolatey to `C:\Chocolatey` or somewhere besides the default. **NOTE**: the root of `C:\` is not a secure location by default, so you may want to update the security on the folder.
```

Comment:
Looks like we lost the security recommendation against installing to the root of the drive here. Any change to bring that back in some way?